cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/utils.js

```diff
@@ -222,6 +222,87 @@ function trackEvent(exchangeID, data, event, customerEmail) {
 }
 
 
+// The subscribeUser func takes the user email & phone number to prep a data object w/ a corresponding emailListID or smsListID (both configured in BM w/ values from the Klaviyo Dashboard)
+// Data is sent to the KlaviyoSubscribeProfilesService API to subscribe users to email or SMS lists.
+function subscribeUser(subscribeToEmail, subscribeToSMS, email, phone) {
+    var logger = Logger.getLogger('Klaviyo', 'Klaviyo.core utils.js - subscribeUser()');
+
+    if (klaviyoServices.KlaviyoSubscribeProfilesService == null) {
+        logger.error('subscribeUser() failed - KlaviyoSubscribeProfilesService is null.');
+        return;
+    }
+
+    var emailListID = Site.getCurrent().getCustomPreferenceValue('klaviyo_email_list_id');
+    var smsListID = Site.getCurrent().getCustomPreferenceValue('klaviyo_sms_list_id');
+
+    var data;
+    var result;
+
+    if (subscribeToEmail && emailListID) {
+        data = {
+            data: {
+                type       : 'profile-subscription-bulk-create-job',
+                attributes : {
+                    list_id       : emailListID,
+                    custom_source : 'Marketing Event',
```

Comment:
Can we use 'SFCC Checkout' as the `custom_source` for both subscribe calls?

Reply:
updating